ios/PurchasesHybridCommon/PurchasesHybridCommon/PrivacyInfo.xcprivacy

```diff
@@ -2,6 +2,8 @@
 <!DOCTYPE plist PUBLIC "-//Apple//DTD PLIST 1.0//EN" "http://www.apple.com/DTDs/PropertyList-1.0.dtd">
 <plist version="1.0">
 <dict>
+	<key>NSPrivacyCollectedDataTypes</key>
```

Comment:
I wonder, it should still pickup the values from the native SDK? Just to make sure those are still propagated correctly.

Reply:
Yes! I checked after adding this and it didn't complain

<img width="987" alt="Screenshot 2024-12-16 at 16 30 15" src="https://github.com/user-attachments/assets/f3213871-ce72-40a6-9322-38ff1d41bdeb" />

I think it's most likely an issue on xcode's parsing of the `PrivacyInfo` files tbh